### DISCUSSION TITLE
关于webDav和本地存储的问题

### DISCUSSION THREAD

**Discussion by minshifu** [crimx/ext-saladict#1507] [Q&A]
请教一下，Saladict的New Word List是否在本地也有存储，而不仅仅是在webDav网络端有保存？如果我想更换现在的webDav服务到另一个服务，那么现有的New Word List会继续保持吗？还是在新的webDav端上从零开始？感谢！

**Comment by crimx** [comment]
这个情况说实话我也不太清楚😂。如果迁移的话可以把文件先拷贝过去新的 WebDAV 上再启用同步。

**Comment by minshifu** [comment]
你是说从一个webdav服务云端拷贝到一个webdav服务云端？我怕不同的webdav服务不能互相兼容吧？  我大概试了一下，目前应该是本地保留一份，云端保留一份，这样挺好。而且插件里也有显示（local only）的部分。只是这个local only的话能确定是哪个文件夹的哪个文件吗？
![图片](https://user-images.githubusercontent.com/5119952/137686966-076dbe76-2138-4d14-b38c-afdbba4d42a1.png)

**Comment by crimx** [comment]
本地和云端都会存一份的，WebDAV 都是这么运作。我是说更换 WebDAV 服务器的时候建议先把 Saladict 文件夹从原来的服务器拷贝到新的上面，然后再在沙拉查词修改设置，这样可以避免丢失问题。

或者你也可以先下载下来，直接更换服务器，出现问题了再覆盖回去。

抱歉当时实现的时候没考虑这个场景，所以不能保证稳定性。

**Reply by minshifu** [reply]
好的，多谢哈！
